frontend/src/app/Routes.js

```diff
@@ -17,10 +18,7 @@ function Routes() {
         <h2>Insert stores list/map</h2>
       </Route>
       <Route exact path="/stores/:id" component={Store} />
-      <Route exact path="/company">
-        <h2>Insert company backend view (falls back to login) if user has a
-          company show edit, otherwise create view</h2>
-      </Route>
+      <Route exact path="/company" component={CompanyPage} />
```

Comment:
Change of heart, can you maybe use the Store component for this? I think its nicer because you don't have to copy that much JSX code then. You can keep the path for now

Reply:
How would I implement this